src/Qir/check-sources-formatted.ps1

```diff
@@ -0,0 +1,31 @@
+#Requires -PSEdition Core
+#Requires -Version 7.1
+
+param (
+    [Parameter()]
+    [String]
+    $DirPath = "$PSScriptRoot"
+)
+
+$tmpFile = "format.log"
```

Comment:
You can explicitly get a temporary file name with ` [System.IO.Path]::GetTempFileName()`.
```suggestion
$tmpFile = [System.IO.Path]::GetTempFileName();
```

Reply:
Thank you, that's good to know.
But for me locally the `[System.IO.Path]::GetTempFileName()` returns `C:\Users\rokuzmin\AppData\Local\Temp\tmp3C56.tmp`
(a path in a different directory, and an unpredictable file name). 

For debugging and failure analysis purposes I'd prefer the file to have a predictable/known path and file name.
I can change the filename to a more unique name (e.g. add a const GUID to the name).